packages/amplify_api/ios/Classes/SwiftAmplifyApiPlugin.swift

```diff
@@ -19,19 +19,24 @@ import Amplify
 import AmplifyPlugins
 
 public class SwiftAmplifyApiPlugin: NSObject, FlutterPlugin {
-
+    
```

Comment:
This file got a lot of whitespace changes

Reply:
I reformatted it using Xcode

Reply:
This can be easily fixed: Xcode->Preferences->Text Editing->Automatically trim trailing whitespace + Including whitespace-only lines and then reformat.